app/services/asset_derivative_storage_type_auditor.rb

```diff
@@ -31,6 +31,9 @@
 class AssetDerivativeStorageTypeAuditor
   attr_reader :incorrect_storage_locations, :incorrectly_published
 
+  # We're storing a short list of sample friendlier_ids of assets with
+  # problems. How many?
+  PROBLEM_SAMPLE_SIZE = 10
```

Comment:
Should we make sure to mention to stakeholders when presenting this that if there's a failure, the report will only tell about at most 10 failed ones? Then I guess they'll have to ask developers for help finding if there are more? I have no opinion on whether this is sufficient, we should just clear it with stakeholders.

Reply:
Oh, I highlighted the recipe in the copy on the form: ```This report confirms that files that should not be accessible to the public are indeed locked down, and vice-versa. To save space, we only list the total number of assets with problems, along with a short sample of actual links to some of those assets. Reports are run nightly, but we only keep the latest one.```

Reply:
And the UI makes it clear that it's not the full list. When there are problems to display, I list the ones we have stored, but followed by an ellipsis, and then the total number of items in parenthesis.

Reply:
OK let's just make sure to bring it up when presenting for review/approval too, since it's easy to miss on a quick skim. They may ask "why'd you do it that way".